.vscode/cspell.json

```diff
@@ -276,6 +278,51 @@
       "words": [
         "essai"
       ]
+    },
+    {
+      "filename": "sdk/textanalytics/azure-ai-textanalytics/**",
+      "words": [
+        "Forndexter",
+        "ESDNI",
+        "dinero",
+        "IOHTTP",
+        "Zocor",
+        "dann",
+        "dont",
+        "UMLS",
+        "nach",
+        "año",
```

Comment:
Unfortunately, the best way to handle the cases of languages that aren't English right now is to add exceptions for the words or ignore the files (like you did here). 

If all of the non-English words are in specific files one could scope the override to just those files but only if excluding those files from spell checking doesn't increase the chance of a spelling error slipping into something a customer might see (e.g. samples, public API interface, etc.)